ui/apps/platform/src/Containers/Search/ViewLinks.tsx

```diff
@@ -25,7 +27,7 @@ function ViewLinks({ id, resultCategory, searchResultCategoryMap }: ViewLinksPro
                             variant="link"
                             isInline
                             component={LinkShim}
-                            href={id ? `${basePath}/${id}` : basePath}
+                            href={safeGeneratePath(basePath, searchResult, basePath)}
```

Comment:
Instead of using `generatePath` directly, use a safe wrapper function that handles the try/catch and uses a fallback value. A code change that modifies the path string or the incoming data object could result in a runtime error *during render* that can not be caught at compile time when using `generatePath` directly.